README.md

```diff
@@ -109,6 +112,13 @@ The following options are available:
      --marketplace-address  Address of deployed Marketplace contract.
      --validator            Enables validator, requires an Ethereum node [=false].
      --validator-max-slots  Maximum number of slots that the validator monitors [=1000].
+     --validator-partition-size  Slot validation partition size [=1].
+                            A number indicating total number of partitions into which the whole slot id
+                            space will be divided. The value must be greater than 0.
+     --validator-partition-index  Slot validation partition index [=0].
```

Comment:
Don't forget to update these with the new names 🙏

Reply:
Good catch 👍